CSS/modules.css

```diff
@@ -0,0 +1,120 @@
+/*******  CSS/MODULES ********/
+
+/*******  HEADER/ NAV  ********/
+
+.header__div__h1 {
```

Comment:
It would be better to select these just with a descendent selector, like
```
.header h1
```
BEM-style selectors aren't exactly used like this--they generally use semantic descriptions instead of base elements, like:
```
primary-menu__menu-item